.github/workflows/build.yml

```diff
@@ -303,7 +417,40 @@ jobs:
             - name: Build the loader
               run: ninja -C out/${{matrix.config}}
 
-    mingw:
+    # MinGW uses JWASM to compile the assembly code - since that isn't available on github actions, this run is disabled
```

Comment:
Is there no way to get JWASM via github actions via some sort of sudo apt get? Or whatever MinGW does?

Reply:
There could definitely be, but I felt getting the current changes out there (to fix the macOS regression) took precedence.